Cargo.toml

```diff
@@ -142,9 +144,10 @@ arbitrary = ["dep:arbitrary", "std"]
 ark-ff = ["dep:ark-ff-03"]
 ark-ff-04 = ["dep:ark-ff-04"]
 bn-rs = ["dep:bn-rs", "std"]
+fastrlp = ["dep:fastrlp-03", "alloc"]
```

Comment:
nit: alphabetize

Reply:
done in https://github.com/recmo/uint/pull/403/commits/947d0f5bcc998f07e0c1bf15f193b31360fd2294